source/installation/install_postgres_server.rst

```diff
@@ -11,16 +11,64 @@
 Install PostgresSql server
 ##########################
 
-.. todo::
+On a linux environment, installing Postgresql server is very easy.
+Here follows the instructions for a debian based distribution (as Ubuntu).
 
-   the section "Install PostresSql database" must be written. Help appreciated :-)
-   
+.. code-block:: bash
+	
+	sudo apt-get update
+	sudo apt-get install postgresql
```

Comment:
It is recommanded to use the `apt` of postgresql. You will have always the last version. Doc here : https://wiki.postgresql.org/wiki/Apt